perf/index.js

```diff
@@ -0,0 +1,47 @@
+const Benchmark = require('benchmark');

```

Comment:
I didn't get exactly what Benchmark will bring to fastcheck but why doesn't this code  need to be tested? ;-)

Reply:
Benchmark is used to check and measure performances gaps. It has to be run and checked manually. It should be considered as a way to detect slow sections of the code.